dev-tools/docker/Dockerfiles/pga_config.php.j2

```diff
@@ -11,88 +11,80 @@ return array(
         /**
          * Admin Role Name
          */
-        'admin-role-name' => '{{ admin_role_name }}',
+        'admin-role-name' => 'admin',
 
         /**
          * Read only Admin Role Name
          */
-        'read-only-admin-role-name' => '{{ admin_readonly_role_name }}',
+        'read-only-admin-role-name' => 'admin-read-only',
 
         /**
          * Gateway user role
          */
-        'user-role-name' => '{{ user_role_name }}',
+        'user-role-name' => 'gateway-user',
 
         /**
          * Initial user role. This is the initial user role assigned to a new
          * user. Set this to one of the three roles above to automatically
          * grant new users that role, or set to some other role ('user-pending')
          * to require admin approval before users have access.
          */
-        'initial-role-name' => '{{ initial_role_name }}',
+        'initial-role-name' => 'user-pending',
 
         /**
          * Tenant Domain
          */
-        'tenant-domain' => '{{ tenant_domain }}',
+        'tenant-domain' => 'test-gateway',
 
         /**
          * Tenant admin's username
          */
-        'admin-username' => '{{ admin_username }}',
+        'admin-username' => 'admin',
 
         /**
          * Tenant admin's password
          */
-        'admin-password' => '{{ admin_password }}',
+        'admin-password' => 'admin',
```

Comment:
I don't think this file with sensitive content should be checked into repository. Although the passwords seem to be for your local installation, it would be better if u revert it to the {{ variable }} version.

Reply:
But these configurations cannot be parameterized as these files directly go into the docker container through Dockerfile. To test the installation, these values are not parameterized.